### DISCUSSION TITLE
v3-beta

### DISCUSSION THREAD

**Discussion by harshraj8843** [codinasion/codinasion-programme#1343] [Announcements]
We are migrating [Codinasion](https://codinasion.web.app/) site from [Gatsby Js](https://www.gatsbyjs.com/) to [Next Js](https://nextjs.org/)

Everyone is welcome to provide their valuable feedback, report any bug, suggest new features and more...

Programme Demo :- https://codinasion.vercel.app/programme

Thanks 💖 

All Feature Requests ( todo list ) will be tracked at https://github.com/orgs/codinasion/projects/2

**Comment by Rebeccaxxxxx** [comment]
Hi, thank you for making this great project! 
I looked at the demo and I feel like having a search/filter function would make using it easier, especially if the number of codes increases in the future!

**Reply by harshraj8843** [reply]
Thanks @Rebeccaxxxxx for your suggestion 👍🏻 

This is really a good idea.

**Comment by harshraj8843** [comment]
add search bar - https://github.com/orgs/codinasion/discussions/3
